src/api/java/com/minecolonies/api/crafting/CountedIngredient.java

```diff
@@ -69,12 +69,16 @@ public CountedIngredient(@NotNull final Ingredient child, final int count)
     @Override
     public IIngredientSerializer<? extends Ingredient> getSerializer()
     {
-        return Serializer.INSTANCE;
+        return Serializer.getInstance();
     }
 
     public static class Serializer implements IIngredientSerializer<CountedIngredient>
     {
-        public static final Serializer INSTANCE = new Serializer();
+        private static final Serializer INSTANCE = new Serializer();

```

Comment:
Yes this is what I want to see, and what I requested.

If there are classes you found which do not follow this pattern, send them to me.
Allthough I would find this very weird, since i have this pattern as a FileTemplate in my IDE, and it would be weird if as such Singletons with other structures were created, but hey, I might have missed them during reviews.

Reply:
`FactoryVoidInput` is the only one that does it with a public `INSTANCE` field, at least.  There may be other cases with different names.  But there are many other places where the `getInstance()` refers to a non-final field.